modules/oc-mirror-IDMS-ITMS-about.adoc

```diff
@@ -4,13 +4,18 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="oc-mirror-custom-resources-v2_{context}"]
-= About custom resources generated by v2
+= About custom resources generated by oc-mirror plugin v2
 
 // Should sentence below say "to which a digest or tag refers"?
 
-With oc-mirror plugin v2, `ImageDigestMirrorSet` (IDMS) resources are generated by default if at least one image of the image set is mirrored by digest.
-`ImageTagMirrorSet` (ITMS) resources are generated if at least one image from the image set is mirrored by tag.
+The oc-mirror plugin v2 automatically generates the following custom resources:
 
-Operator Lifecycle Manager (OLM) uses the `CatalogSource` resource to retrieve information about the available Operators in the mirror registry.
+`ImageDigestMirrorSet` (IDMS) resources:: Handles registry mirror rules when using image digest pull specifications. Generated if at least one image of the image set is mirrored by digest.
 
-The OpenShift Update Service uses the `UpdateService` resource to provide update graph data to the disconnected environment.
\ No newline at end of file
+`ImageTagMirrorSet` (ITMS) resources:: Handles registry mirror rules when using image tag pull specifications. Generated if at least one image from the image set is mirrored by tag.
+
+`CatalogSource` resources:: Retrieves information about the available Operators in the mirror registry. Used by {olmv0-first}.
+
+`ClusterCatalog` resources:: Retrieves information about the available cluster extensions (which includes Operators) in the mirror registry. Used by {olmv1}.
+
+`UpdateService` resources:: Provides update graph data to the disconnected environment. Used by the OpenShift Update Service.
```

Comment:
Is having the word _resources_ in the list redundant, as the term is in the introductory line? Also eliminates the odd plural noun:: singular verb thing.

```suggestion
`ImageTagMirrorSet` (ITMS):: Handles registry mirror rules when using image tag pull specifications. Generated if at least one image from the image set is mirrored by tag.

`CatalogSource`:: Retrieves information about the available Operators in the mirror registry. Used by {olmv0-first}.

`ClusterCatalog` :: Retrieves information about the available cluster extensions (which includes Operators) in the mirror registry. Used by {olmv1}.

`UpdateService`:: Provides update graph data to the disconnected environment. Used by the OpenShift Update Service.
```